template2.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
```

Comment:
# [OPTIONAL]
- [ ] I see that you did all sections but you did it apart from the original mobile work (the header and the hello section). Consider putting all on the same HTML page.